_posts/2016-06-07-why-socrates-uk-is-not-just-a-conference-for-software-craftsmen.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+name: Why-SoCraTes-UK-is-not-just-a-conference-for-Software-Craftsmen
+title: 'Why SoCraTes UK is not just a conference for Software Craftsmen.'
+date: 2016-06-07 15:20:00 +00:00
+author: Giulia Mantuano
+image:
+    src: /assets/img/custom/blog/socrates.jpg
+tags:
+- socrates
+- conference
+- craftsmanship
+---
+
+Last Friday, I attended the international [Software Craftsmanship and Testing conference](http://socratesuk.org/index.html) at the enchanting Wotton House, in Dorking. I was the only Designer among about 100 Developers.
+
+Honestly, I was feeling a little bit out of my element when I arrived, but I knew that this was going to be a great opportunity to learn more about my team members and dispel the myth about the difficult relationship between designers and developers.
+
+While the attendees were sharing their knowledge about TDD, Domain Driven Design, Event Sourcing, Functional Programming, all topics already familiar to the community, I decided to propose a one-hour session about the 5 principles of Interaction Design.
+It was really important to me to share something that could help an audience with little or no design background to craft better interfaces in the future.
```

Comment:
I don't know exactly the grammar rule about the difference between "Important to sb"/"Important for sb", but it seems fine using "to"